### ISSUE TITLE
Unexpected warning, unsupported browsers being included

### ISSUE THREAD

**Issue by magicspon** [amilajack/eslint-plugin-compat#250]
I'm getting the following warning:

`FileReader is not supported in Safari 5.1eslint(compat/compat)`

In my package.json file I have the following:

```json
"browserslist": [
	"safari 9",
	"ie 11",
	"> 0.5%"
]
```

Which should not include safari 5.1

https://browserl.ist/?q=safari+9%2C+ie+11%2C+%3E+0.5%25

Running npx browserslist, produces the following:
```
and_chr 75
and_uc 11.8
chrome 75
chrome 74
chrome 73
edge 18
edge 17
firefox 67
ie 11
ios_saf 12.2-12.3
ios_saf 12.0-12.1
ios_saf 11.3-11.4
op_mini all
opera 60
safari 12.1
safari 9
samsung 9.2
samsung 4
```

Any ideas on how i resolve this?

Thanks

**Comment by richmidd91** [comment]
Not sure what your setup is, but I had the same issue with my codebase. My issue was I had multiple submodules, and needed to update the .browserslistrc for each submodule.

**Comment by aMarCruz** [comment]
New versions of browserslist include Safari 5.1 by default.
Exclude it explicitly or use `safari >= 9`

I recommend you include `"browsers": "browserslist"` in the "scripts" block of your package.json, then enter `yarn add browserslist -D && yarn browsers`.

... or `npm i browserslist -D && npm run browsers`, do not use `npx` for this.
